ppx/test/input_ocaml.ml

```diff
@@ -1,24 +1,18 @@
-[@@@react.dom]
+let%component make ?(name = "") = div [||] [React.string ("Hello " ^ name)]
 
-let%component make ?(name = "") =
-  div ~children:[React.string ("Hello " ^ name)] ()
+let element = div [|id "foo"|] []
 
-let element = div ~id:"foo" ~children:[] ()
+let element = a [|id "foo"; href "https://www.hello.com"|] [React.string "hello"]
```

Comment:
I would think these can be removed, as there's no syntax transformation going on here anymore.